traits-stubs/traits-stubs/trait_types.pyi

```diff
@@ -0,0 +1,664 @@
+from pathlib import PurePath as _PurePath
+from typing import (
+    Any as _Any,
+    Callable as _CallableType,
+    Dict as _DictType,
+    List as _ListType,
+    Optional,
+    Sequence as _Sequence,
+    Set as _SetType,
+    Type as _Type,
+    TypeVar,
+    Union as _Union,
+)
+from uuid import UUID as _UUID
+
+from .trait_type import _TraitType
+
+MutableTypes: _Any
+SetTypes: _Any
+int_fast_validate: _Any
+float_fast_validate: _Any
+complex_fast_validate: _Any
+bool_fast_validate: _Any
+
+
+def default_text_editor(trait: _Any, type: Optional[_Any] = ...):
+    ...
+
+
+_T = TypeVar("_T")
+_S = TypeVar("_S")
+
+_Trait = _Union[_TraitType[_S, _T], _Type[_TraitType[_S, _T]]]
+
+
+class Any(_TraitType[_Any, _Any]):
```

Comment:
I think we might need to do something like `Any = _Any` because
```
class Superclass(HasTraits):
    x = Any()
    y = Any()

class Subclass(Superclass):
    x = Instance(Foo)
    y = Int()
```
doesn't work with the current definition.

Reply:
Looking at the mypy codebase, `mypy.subtypes.is_subtype`, the docstring does mention `Any` will be considered as a subtype of any type and vice versa, but I couldn't get it work either. Guess I'll go with using the alias `Any = _Any`